specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironmentsStorages.json

```diff
@@ -0,0 +1,321 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-01-01-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{envName}/storages": {
+      "get": {
+        "tags": [
+          "ManagedEnvironmentsStorages"
+        ],
+        "summary": "Get all storages for a managedEnvironment.",
+        "description": "GGet all storages for a managedEnvironment.",
+        "operationId": "ManagedEnvironmentsStorages_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "envName",
+            "in": "path",
+            "description": "Name of the Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagedEnvironmentStoragesCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List environments storages by subscription": {
+            "$ref": "./examples/ManagedEnvironmentsStorages_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{envName}/storages/{name}": {
```

Comment:
>storages [](http://example.com/codeflow?start=126&length=8)

consider updating the name. "storages" is not really a word. Something like "stores" or "managedstores" (These are just suggestions please work with your team to settle on a proper name)

Reply:
https://en.wiktionary.org/wiki/storages#Englis
https://www.wordhippo.com/what-is/the-plural-of/storage.html

I think storage have a "plural" as "storages". What do you think?